sdk/storage/azure-storage-blob/azure/storage/blob/_shared_access_signature.py

```diff
@@ -299,7 +318,7 @@ def add_resource_signature(self, account_name, account_key, path, user_delegatio
                         sign_string(account_key if user_delegation_key is None else user_delegation_key.value,
                                     string_to_sign))
 
-    def get_token(self):
+    def get_token(self) -> str:
```

Comment:
Again, you only typed one method of many in this class. Either fill in types for the whole class or revert this (since its private).